ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-stable-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -80,6 +80,46 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-upgrade-qe-sanity
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
+- as: aws-ipi-proxy-sts-arm-f28
```

Comment:
Sorry I did not get the point, do you mean https://github.com/openshift/release/blob/a60ce7507ac9ea61a5eb2a1ad97158aa491f6fd0/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml#L440 ?

Reply:
yes

Reply:
It is in 4.15__multi_nightly, do I miss something ?

Reply:
I mean related e2e job **only** exists in 4.15__multi-nightly.yaml, but now you are adding it in 4.15__multi-**stable**-4.15-upgrade-from-stable-4.14.yaml, does it make sense?

Reply:
Sorry, I should type more words in comment...

Reply:
The job exists in source **4.14** nightly ci.
```
openshift-openshift-tests-private-release-4.14__multi-nightly.yaml:- as: aws-ipi-proxy-sts-arm-f28
openshift-openshift-tests-private-release-4.14__multi-nightly.yaml:- as: aws-ipi-proxy-sts-amd-f28-destructive
```

Reply:
Before the job exists in amd and arm stable upgrade ci, e.g: https://github.com/openshift/release/pull/55554/files#diff-c4bc9e4f5ea515dab9026df4b8cc55c3e8dba20df88a4b8cd7cd7f6c03f12d11L97, now I am migrating them to multi-arch stable upgrade ci, not adding new ones.

Reply:
Sorry, correct my words:
I mean related e2e job only exists in 4.1**4**__multi-nightly.yaml, but now you are adding it in 4.15__multi-stable-4.15-upgrade-from-stable-4.14.yaml, does it make sense?

IMO, we should add nightly e2e job to nightly upgrade, and stable e2e to stable upgrade.

Reply:
Just like I explained in the above comment, the reason of not existing in nightly upgrade ci, that is because we missed the profile in nightly upgrade ci. While in this PR, I am focus on stable upgrade ci tidy up, the profile exists arm/amd stable upgrade ci for a long time, I just migrate them to mult-arch stable ci. The missing profiles on nightly upgrade ci is out of the PR scope.

Reply:
Wait a sec, I double check the nightly upgrade ci, the profile is being covered.
```
openshift-openshift-tests-private-release-4.15__multi-nightly-4.15-upgrade-from-stable-4.14.yaml:- as: aws-ipi-proxy-sts-arm-f14
openshift-openshift-tests-private-release-4.15__multi-nightly-4.15-upgrade-from-stable-4.15.yaml:- as: aws-ipi-proxy-sts-arm-f28
```

Reply:
So we actually do not need to cover it here, right?

Reply:
We need it, the job is in mult-arch stable upgrade ci, before it was in arm/adm signle-arch stable upgrade ci, now I migrated it to mult-arch ci.